components/dialog/dialog-fullscreen.js

```diff
@@ -226,9 +210,6 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 				: 0;
 			const startTop = mediaQueryList.matches ? 42 : 0;
 			topOverride = iframeTop + startTop;
-		} else if (window.innerWidth <= 615 || (window.innerWidth <= 900 && window.innerHeight <= 420)) {
-			heightOverride.height = `${window.innerHeight - 42 - 2}px`; // render full window height - 42px top padding - 2px border
```

Comment:
This change here is going to result in a `2px` increase in height for all fullscreen dialogs. As is written in the comment, this takes the window height and subtracts `44px`, whereas the CSS solution is just the viewport height minus `42px`. I think this is an OK outcome to align things.

Reply:
Looks like this was recently added to fix [an issue on mobile](https://github.com/BrightspaceUI/core/pull/2200).  I don't know much about that change but just a heads up!

Reply:
You're right, this does reintroduce that problem... so weird that normal dialogs don't exhibit it now that they're essentially the same. Back to the drawing board I guess!

Reply:
Ok fixed! We had a viewport-size helper that defined `--d2l-vh` but it needed to actually be included on the page in the case of fullscreen dialogs to work. :)

Thanks for catching that Stacey.